python/ml4ir/base/features/feature_config.py

```diff
@@ -368,6 +368,40 @@ def get_all_features(self, key: str = None, include_label: bool = True):
                     if fdict["name"] != self.get_label("name")
                 ]
 
+    def get_all_values(self, key: str = None, include_label: bool = True):
```

Comment:
@balikasg I'm trying to understand what is the need for this. Are you trying to use this get all the vocab files at once?

Reply:
yes. exactly.